### PR TITLE
changes for pcm-519

### PR DESCRIPTION
**Pull request by pbathia**

### CODE REVIEW

**Comment by pbathia** [comment]
@renujhamtani @sshumake Please review and merge.

File: app/ascension/views/caseSearch.jade [hunk 1]
```diff
@@ -1,6 +1,5 @@
-.col-md-9
```
Comment (pbathia) [inline app/ascension/views/caseSearch.jade]: This was causing case search input box to appear in middle as Renu removed 'Your Cases' with col-md-3 in her last PR.
I just corrected it, and not related to PCM-519.

**Comment by renujhamtani** [comment]
Reviewed! @ardasgup Can u please review?

File: app/ascension/services/caseDetailsService.js [hunk 2]
```diff
@@ -123,6 +124,16 @@ angular.module('RedhatAccess.ascension').service('CaseDetailsService', [
                         self.kase.observers.push(associate.resource.associate.resource.fullName);
                     }
                 });
+                self.isLoggedInUserIsOwner = false;
+                //when case owner is 'New Case Queue' , we dont get kase.owner field from UDS
+                if(RHAUtils.isNotEmpty(this.kase.owner)) {
+                    //we have to iterate for sso, as one user can have multiple sso in the UDS response
```
Comment (ardasgup) [inline app/ascension/services/caseDetailsService.js]: @pbathia do you know why this is so ? one user having multiple sso?

**Comment by ardasgup** [comment]
Reviewed!

File: app/ascension/services/caseDetailsService.js (hunk 2 above)
Comment (pbathia) [inline app/ascension/services/caseDetailsService.js]: (06:08:50 PM) pbathia: dkoul: why we have multiple sso for single user ?
(06:10:46 PM) dkoul: pbathia, people demand :)
(06:10:55 PM) pbathia: dkoul: ok 
(06:11:11 PM) dkoul: pbathia, one sso per user BUT multiple sso's per human

Comment (ardasgup) [inline app/ascension/services/caseDetailsService.js]: @pbathia :)

**Pull request merged**